src/test/java/Homework17.java

```diff
@@ -0,0 +1,77 @@
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest {
+    //extends BaseTests - inherits the methods /attributes of basetest class
+
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._